filters/ThirdParty/filter_249_NorwegianList/metadata.json

```diff
@@ -11,6 +11,9 @@
   "tags": [
     "purpose:ads",
     "purpose:security",
-    "lang:no"
+    "lang:no",
+    "lang:da",
+    "lang:is",
+    "lang:fo"
```

Comment:
This one needs to be added to https://github.com/AdguardTeam/FiltersRegistry/blob/master/tags/metadata.json

Reply:
Also, if you don't mind, add "recommended" tag here as well.

Reply:
Presuming that you want me to take care of that, I'll get around to it right away. 🙂

Reply:
Also, every tag has a name and description: https://github.com/AdguardTeam/FiltersRegistry/blob/master/locales/en/tags.json